manifests/v1alpha2/katib-controller/katib-controller.yaml

```diff
@@ -18,8 +18,8 @@ spec:
       serviceAccountName: katib-controller
       containers:
       - name: katib-controller
-        image: katib/katib-controller
-        imagePullPolicy: Always
+        image: katib/v1alpha2/katib-controller
```

Comment:
you need modify https://github.com/kubeflow/katib/blob/master/test/scripts/v1alpha2/run-tests.sh#L68-L71 to match the change

Reply:
SGTM. I forgot to do it.